samza-rest/src/main/java/org/apache/samza/rest/resources/ResourceConstants.java

```diff
@@ -20,7 +20,7 @@
 
 public class ResourceConstants {
 
-  public static final String GET_TASKS_URL = "http://%s/%s/%s/tasks/";
+  public static final String GET_TASKS_URL = "http://%s/v1/jobs/%s/%s/tasks/";
```

Comment:
Behavior was tested and verified initially within a LI cluster. 

However, there were problems with merging the independent patches  where in the  url format of endpoint got changed and the actual client wasn't updated properly.